src/engraving/dom/drumset.h

```diff
@@ -76,7 +76,7 @@ struct DrumInstrument {
     bool operator==(const DrumInstrument& other) const
     {
         return notehead == other.notehead
-               && noteheads == other.noteheads
+               // && noteheads == other.noteheads // TODO: Investigate why this is can be unexpectedly false
```

Comment:
Maybe because this performs pointer comparison rather than array contents comparison? 
If so, perhaps `std::array` can help.

Reply:
Oh yes that'll be it - thanks Casper!